x-pack/plugins/security_solution/public/detection_engine/rule_creation/components/esql_info_icon/index.tsx

```diff
@@ -30,14 +27,20 @@ const EsqlInfoIconComponent = () => {
 
   return (
     <EuiPopover button={button} isOpen={isPopoverOpen} closePopover={closePopover}>
-      <EuiText style={{ width: POPOVER_WIDTH }} size="s">
-        <Markdown
-          markdown={i18n.getTooltipContent(
-            docLinks.links.esql.statsBy,
-            // Docs team will provide actual link to a new page before release
-            // For now, it's just a mock
-            docLinks.links.esql.statsBy
-          )}
+      <EuiText size="s">
+        <FormattedMessage
+          id="xpack.securitySolution.detectionEngine.createRule.stepDefineRule.esqlInfoTooltipContent"
+          defaultMessage="Check out our {createEsqlRuleTypeLink} to get started using ES|QL rules in Security."
```

Comment:
Since users are already in the Security app, I don't think we need to specify that this is a Security rule. Also, is there any way to remove the icon after "documentation"? It looks a little out of place with the inline link.

```suggestion
          defaultMessage="Check out our {createEsqlRuleTypeLink} to get started using ES|QL rules."
```

Reply:
this seems to be a standard way of outlining external links: https://eui.elastic.co/#/navigation/link#external-links

Reply:
Ah, I didn't see this page that shows how to set up inline links. I was looking at these guidelines instead: https://eui.elastic.co/#/guidelines/writing/examples#links

Anyways, no worries if it's not possible to remove the icon after the linked text. The guidelines that I was looking at made it look like you could, so I figured I'd ask.

Reply:
I think it's possible, but the link will open new window, so that's why icon there, to inform that user will leave current tab

Reply:
Gotcha - thanks for elaborating!